### PR TITLE
Visualisation Blog

### PR DESCRIPTION
**Pull request by Rucknar**
New Blog for the site. Happy for anyone to review/merge.

### CODE REVIEW

File: _posts/2016-10-13-contributing.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+We're off to a good start, but it's time to get our hands dirty to get at the information we need. Neither Github nor Docker Hub have native Prometheus support, there also wasn't an option we could find within the community.
+We set out to create some tools for us and others to use, should they find them useful... (yay opensource!) 
+
+* Github Exporter ([Docker Hub](https://hub.docker.com/r/infinityworks/github-exporter/), [GitHub](https://github.com/infinityworksltd/github-exporter))
+* Docker Hub Exporter ([Docker Hub](https://hub.docker.com/r/infinityworks/docker-hub-exporter/), [GitHub](https://github.com/infinityworksltd/docker-hub-exporter))
+
+Whilst simple, the above two exporters helps us capture the metrics from the APIs and present them for scraping by the prometheus server. We can use these metrics to measure our contribution back to the community.
+
+Github fortunately has a simple, well documented API to retrieve the information you need. We made us of the Prometheus client libraries and some python to extract the data we needed from repositories and organisations.
```
Comment (a-h) [inline _posts/2016-10-13-contributing.md]: Fortunately, Github has a simple, well documented API to retrieve the information we needed. We made use of the Prometheus client libraries and some python to extract the data we needed from repositories and organisations.

File: _posts/2016-10-13-contributing.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+First, we have an overview of our Github activity, click [here]({{ site.baseurl }}/images/github_stats.png) to view fullscreen.
+![GitHub Screenshot]({{ site.baseurl }}/images/github_stats.png "GitHub Screenshot")
+
+Secondly, we have an overview of our DockerHub activity, click [here]({{ site.baseurl }}/images/docker_hub_stats.png) to view fullscreen.
+
+![Docker Hub Screenshot]({{ site.baseurl }}/images/docker_hub_stats.png "Docker Hub Screenshot")
+
+## What's next?
+
+This is very much a first step in the direction of what we hope can be achieved. We would like to compliment the data gathered with richer metrics, from more sources. How many Slack messages did we exchange today? How many cups of coffee did the team last month? The list is endless, but we've made a start. And of course, it's all open-source. We encourage you to get involved and improve upon what's there so far.
```
Comment (a-h) [inline _posts/2016-10-13-contributing.md]: compliment -> complement

File: _posts/2016-10-13-contributing.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: Visualising Our Digital Footprint
+author: ed_marshall
+image: visual_header.jpeg
```
Comment (a-h) [inline _posts/2016-10-13-contributing.md]: Do we have permission to use this image? Where does it come from?

Comment (Rucknar) [inline _posts/2016-10-13-contributing.md]: It's a license free no attribution image, clear from any usage concerns.

File: _posts/2016-10-13-contributing.md (hunk 2 above)
Comment (Rucknar) [inline _posts/2016-10-13-contributing.md]: Updated

**Pull request merged**